crypto/property/property_parse.c

```diff
@@ -453,6 +453,33 @@ int ossl_property_has_optional(const OSSL_PROPERTY_LIST *query)
     return query->has_optional ? 1 : 0;
 }
 
+int ossl_property_is_enabled(OPENSSL_CTX *ctx,  const char *property_name,
+                             const OSSL_PROPERTY_LIST *prop_list)
+{
+    int i;
+    OSSL_PROPERTY_IDX name_id;
+    const PROPERTY_DEFINITION *prop = NULL;
+
+    if (prop_list == NULL)
+        return 0;
+
+    if (!parse_name(ctx, &property_name, 0,&name_id))
+        return 0;
+
+    prop = prop_list->properties;
+    for (i = 0; i < prop_list->n; ++i) {
+        if (prop[i].name_idx == name_id) {
+            return prop[i].optional
```

Comment:
optional is a match?  [see note below]

Reply:
Changed to !optional && ()

Reply:
Ok this change combined with the -fips setting resulted in a nasty side effect..
It now also needs to check for override since the type is not set for these - and will result in uninitialized reads if accessed.